src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -232,13 +232,13 @@ class WorkspaceMembersPage extends React.Component {
                         />
                     </View>
                     {this.props.session.email === item.login && (
-                        <View style={styles.peopleRowCell}>
-                            <View style={[styles.badge, styles.peopleBadge]}>
-                                <Text style={[styles.peopleBadgeText]}>
-                                    {this.props.translate('common.admin')}
-                                </Text>
-                            </View>
+                    <View style={styles.peopleRowCell}>
+                        <View style={[styles.badge, styles.peopleBadge]}>
+                            <Text style={[styles.peopleBadgeText]}>
+                                {this.props.translate('common.admin')}
+                            </Text>
                         </View>
+                    </View>
```

Comment:
why the indentation change? was it wrong? I think it is wrong now

Reply:
Was not on purpose, this is from when I reverted the changes in this file.